src/renderer/controllers/torrent-list-controller.js

```diff
@@ -182,47 +182,51 @@ module.exports = class TorrentListController {
 
   openTorrentContextMenu (infoHash) {
     const torrentSummary = TorrentSummary.getByKey(this.state, infoHash)
-    const menu = new electron.remote.Menu()
-
-    menu.append(new electron.remote.MenuItem({
-      label: 'Remove From List',
-      click: () => dispatch('confirmDeleteTorrent', torrentSummary.infoHash, false)
-    }))
-
-    menu.append(new electron.remote.MenuItem({
-      label: 'Remove Data File',
-      click: () => dispatch('confirmDeleteTorrent', torrentSummary.infoHash, true)
-    }))
-
-    menu.append(new electron.remote.MenuItem({
-      type: 'separator'
-    }))
-
-    if (torrentSummary.files) {
-      menu.append(new electron.remote.MenuItem({
+    const torrentStarted = !!(torrentSummary.files && torrentSummary.progress)
```

Comment:
Hmmm, I realise this wouldn't work as `progress` isn't saved in the state. Will try to figure out something else.